container_linux_test.go

```diff
@@ -442,3 +442,80 @@ func TestContainerUsername(t *testing.T) {
 		t.Errorf("expected squid uid to be 31 but received %q", output)
 	}
 }
+
+func TestContainerUserID(t *testing.T) {
+	t.Parallel()
+
+	client, err := newClient(t, address)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer client.Close()
+
+	var (
+		image       Image
+		ctx, cancel = testContext()
+		id          = t.Name()
+	)
+	defer cancel()
+
+	if runtime.GOOS != "windows" {
```

Comment:
nit: not need, this is a linux only test

Reply:
Hm, copied from previous test. Will remove them both.